CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+Please note we have a [code of conduct](https://microsoft.github.io/codeofconduct/), please follow it in all your interactions with the project.
+
+## Pull Request Process
+1. Fork repository and create your feature branch out of main.
+2. Open [Codegen project](./package/Codegen/Codegen.csproj) in Visual Studio.
+3. Right click on project in solution explorer -> select "Manage Nuget Packages". This should open the NuGet Manager which will prompt you to restore NuGet packages. Click on "Restore". Once you restore the packages, the project is ready to build / run.
```

Comment:
this shouldn't be needed (building the CodeGen project for the first time will restore the NuGet packages)